nightwatch.conf.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+
+module.exports = (function() {
+    const settings = {
+        "src_folders": [
+            path.resolve(process.cwd(), "e2e")
+        ],
+        "output_folder": "reports",
+        "custom_commands_path": "",
+        "custom_assertions_path": "",
+        "page_objects_path": "",
+        "globals_path": "",
+        "selenium": {
+            "start_process": true,
+            "server_path": require.resolve('selenium-standalone/.selenium/selenium-server/3.5.3-server.jar'),
+            "log_path": "",
+            "port": 1337,
```

Comment:
Should we change the port to something else, since 1337 was the first number that came to my mind for the test run? Could it be used by something else?

Reply:
We might defer to the default one - 4444, examples are running on 8888 so there will be no collision there.